Korath campaign: first contact, establish lore, side story, and Korath-protected smuggler's haven (#1)
HumanikaRafeki/endless-sky: data/korath/korath missions.txt

	on complete
		payment 1000
		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`

mission "First Contact: Korath Exiles"
	landing

> 	source "Far'en Lai"

Reviewer: ```suggestion
	source "Far'en Lai"
	to offer
		not "Wanderers: Sestor: Exiles 2: active"
```
Probably shouldn't have this pop up when you've got Rek with you.

Author: I think MZ and @ravenshining wrote mission before there was Wanderers middle. There are quite a few problems with it.

Reviewer: That mission is a part of Wanderers Middle, since January 2017. It's a pretty integral part to that string, making contact with the Exiles to get help with the Kor Sestor. It may get reworked/cleared from the game for any Wanderers changes, but for now, it's been a part of the game for over 6 years.

Author: Yes, I'm just saying it isn't the only ancient problem in the mission.

Author: I have merged your suggestion. I don't know why I'm still seeing the "Commit suggestion" button, but I can assure you that your fix is in there.

					goto name
				`	"Oi, oi!"`
					goto silence
				`	"I come in peace. Take me to your leader."`
					goto peaceleader
				`	(Remain silent and observe.)`
					goto silence
				`	(Attack them.)`
					goto attack

			label name
			`	"Takra," it says, pointing to itself. Then it points to the other and says, "Akafavah."`
			`	"Korath," it says, gesturing to the two of them. Then it looks at you expectantly.`
				goto humanika

			# With a longer starting sentence, they suspect you're speaking human standard
			# (even if you may not be human) so they try to use the same language in response.
			label peaceleader
			`	Both of them seem confused, like they half-understand parts of what you said, but can't put the pieces together.`
			`	"Peeese? Takra," it says, pointing to itself. Then it points to the other and says, "Akafavah." You respond almost instinctively with "<first> <last>."`
			`	"Weee Korath," it says, gesturing to the two of them. Then it gestures at you and says, "Yee ess...?"`
				goto humanika

			label humanika
			`	"Human," you say, almost instinctively.`
			`	Akafavah says, "Oy! Humanika!" It's not clear what emotion they're feeling, but it's a strong one.`

> 			`	Takra asks you, "were foam?"`

Reviewer: ```suggestion
			`	Takra asks you, "Were foam?"`
```

				`	(Continue.)`
			`	"Lasel," says the Korath, and it stretches out both arms toward you, fingers curled back towards its body. You make the same gesture, and they reverse their hands, linking their scaly fingers with yours. You're not sure if that is the right response, but the Korath seem satisfied, saying, "Nik lak'mal." They get back in their fighters and take off, cirlcling above you.`
				accept

			label alone
			`	On the map, you draw a line between Korath space and human space. "Humans will stay here," you say, "if the Korath will stay here. Humans here, Korath here." You point to each location for emphasis.`
				goto parting
			label fight
			`	It pushes its finger deep into the ground at a point on the map not too far from Earth. "Ka puk soru, nik ka korath."`
			label parting
			`	"Kai," says the Korath, and it stretches out both arms toward you, palms out. You make the same gesture and touch its hands, palm to palm. You're not sure if that is the right response, but the Korath seem satisfied, saying, "Nik lak'mal." They get back in their fighters, take off, and hover nearby.`
				decline

mission "Korath [F01]: A Fishy Deal"
	landing
	name "A fishy proposition"

> 	description "Transport <cargo> to <planet> by <date>. After a return trip, you'll be payed 140,000. Don't get caught."

Reviewer: ```suggestion
	description "Transport <cargo> to <planet> by <date>. After a return trip, you'll be paid 140,000. Don't get caught."
```

			`	"If the authorities won't let you land, read them the serial number from the cargo container, and they'll open a bay for you. Oh, also, this is an illegal mission, so don't get caught, or it'll cost you. Well, it's unlikely anyone out here will care about laws, but I do believe in full disclosure."`
				accept

	on enter Atik
		dialog "It seems you do not have automatic clearance to <planet>. If you talk to the planet, the authorities will probably let you land."
	on visit
		dialog "You've arrived at <planet> but you left some of the fish behind. You should take off and wait for your escorts to arrive."

event "know about Korath fish jobs"

mission "Korath [F02]: A Fishy Proposition"
	landing
	name "A fishy proposition"
	description "Transport <cargo> to <planet> by <date> for <payment>. Avoid getting scanned or landing on high-security planets, or you may be fined."
	cargo "planetary supplies" 15
	illegal 50000
	"apparent payment" 140000
	deadline 4
	clearance "The authorities do not respond, but a concealed hangar bay door opens. The meaning is clear: land here."
	source "La'ab Bamen"

> 	destination "New Tortuga"

Reviewer: ```suggestion
	destination "New Tortuga"
	to offer
		has "Korath [F01]: A Fishy Deal: done"
```

			label questions
			choice
				`	"That's an interesting ship."`
					goto ship
					to display
						not "Korath [F01]: A Fishy Deal: ship"
				`	"What's getting you down?"`
					goto down
					to display
						has "Korath [F01]: A Fishy Deal: ship"
						not "Korath [F01]: A Fishy Deal: down"
				`	"What sort of delivery?"`
					goto mission
					to display
						has "Korath [F01]: A Fishy Deal: down"
						not "Korath [F01]: A Fishy Deal: mission"
				`	"Sure, I could deliver it for you."`
					goto yup
					to display
						has "Korath [F01]: A Fishy Deal: mission"
				`	"I wish I could help, but I have urgent matters of my own."`
					decline
					to display
						or
							has "Korath [F01]: A Fishy Deal: ship"
							has "Korath [F01]: A Fishy Deal: down"
							has "Korath [F01]: A Fishy Deal: mission"
				`	(Ignore her and go about your business.)`
					decline
					to display
						not "Korath [F01]: A Fishy Deal: ship"
						not "Korath [F01]: A Fishy Deal: down"
						not "Korath [F01]: A Fishy Deal: mission"
			label ship
			action
				set "Korath [F01]: A Fishy Deal: ship"

> 				log "Minor People" "Jenny Jetmaster" "Latest in a family line that has operated and maintained a replica of an ancient jet aircraft and made money on airshows. She is now on run from the Syndicate for rescuing her persecuted uncle from Tinker"

Reviewer 2: ```suggestion
				log "Minor People" "Jenny Jetmaster" "Latest in a family line that has operated and maintained a replica of an ancient jet aircraft and made money on airshows. She is now on the run from the Syndicate for rescuing her persecuted uncle from Tinker"
```

			`	"Poor Jenny. Yes, I know about that. She's smuggling her mother from Freedom to a hospital on Shroud for treatment. Her mother's a strong woman, and I'm sure she'll pull through.`
			`	"She vouched for you, and we did a background check. You do seem suitable for our needs, if you keep a level head. Take these supplies to New Tortuga," she points to a pile of black barrels, "and an officer there will pay you."`
			choice
				`	"I'll see it done."`
					goto see
				`	"You can count on me."`
				`	"No, I think I'm done with your deliveries."`
					goto reject
			`	[Picture of ships in a hovering image near Iris projected by a gadget she's holding.]`
			`	"I'm sure I can," she says, smiling.`
				goto warning
			label see
			`	[Picture of ships in a hovering image near Iris projected by a gadget she's holding.]`
			`	She nods and gives an approving smile.`
			label warning
			`	"One more matter, Captain <last>: a fair warning about some of our associates. To keep helping our planet, you need to be able to travel far and wide, acting as a courier through space we cannot reach. If you anger half the galaxy, not only will it risk your life, it'll hurt the people of New Tortuga.`

> 			`	"You may have noticed some unusual ships near here, larger than anything else in human space. Any fish delivery jobs at New Tortuga, and any jobs at this station, will earn you favor with them. Eventually, they'll stop being hostile, and you could assist their disabled ships.`

Reviewer 2: ```suggestion
			`	"You may have noticed some unusual ships near here, larger than anything else in human space. Any fish delivery jobs at New Tortuga, and any jobs at this station, will earn you favor with them. Eventually, they'll stop being hostile, and you will be allowed to assist their disabled ships.`
```

> 			`	"For your safety, for their sake, and for the sake of my planet, I do implore you: do not assist those ships. They know the risk they take when raiding. It is more important to them that you retain your freedom of movement by not revealing your association."`

Reviewer 2: ```suggestion
			`	"For your safety, for their sake, and for the sake of my planet, I do implore you: do not assist those ships. They know the risk they take when raiding, and have backups in place. It is more important to them that you retain your freedom of movement by not revealing your association."`
```

I don't think that reveals anything concrete or sensitive; and it portrays the Korath in a better light, too. (Not so dumb as to raid without plans...)

Author: They do raid without plans, though, but I hope to fix that in other PRs.

Reviewer 2: Mechanically, yes. They're just random things that occur on a somewhat random basis. Which is fine. But the lore doesn't have to stipulate that they are actually random. Only that they appear to be random to First last.

			`In this trip, your ship's descent stirred up awe-inspiring storm clouds, but traveling through them was easy. Your shield plow technique took you down to the lower troposphere, where the storms cleared enough for you to see. The original island chain was too far away, so you set course for a larger island nearby.`
			`	Aiming for a hard-looking piece of ground, you touch down. As this planet has twice Earth's gravity, you extend your ship's artificial gravity field to the surface. Donning a space suit for safety's sake, you walk out to see nature taking control once again: the massive volcano in front of you starts to erupt.`
			`	After a dash to the cockpit, still in your suit, you take off, satisfied that you've reached the surface - albeit in the worst possible spot. A third attempt will certainly succeed, as you saw a small continent with no apparent volcanic activity, and a pleasant-looking bay.`
				launch

			label "third attempt"
			scene scene/blue-whale
			action
				set "landed on Rabasot Pa'ah"

> 				log "Discovered a creature in a bay on the ocean world of Rabasot Pa'ah with the front half of a whale and the back half of a giant squid. They seem out-of-place, since the planet is almost devoid of macroscopic lifeforms. Perhaps someone is shipping them to the planet?"

Reviewer 2: This might be assuming a bit too much biology/ecosystem knowledge for our lowly textile worker-turned-captain. It's not like we have a detailed scan of the entire planet's ecosystems, either. (likewise for commentary about the whale-squid).

The whale-squid is interesting, but in terms of detail it might be better to focus on what our pilot can see; and less on speculative biology. As far as that goes, something like this giant whale-squid would likely be an apex creature that doesn't have any natural predators or competitors. Likewise, we have the example of the blue whale that feeds on tiny krill. So it is entirely possible that for some reason no other particularly large creatures evolved (or survived).

Author: I will remove the later two sentences, and add something about how difficult it was to get there.

Author: The planet now has a landing pad and a gravity lift device that is large enough to lift one of the creatures. Captain Last's knowledge of evolutionary biology is gone.

			`	"The imperialist Drak race sent living weapons, the Archons, to conquer the Korath under the guise of protection. The Archons' antimatter weapons and seeming invincibility was too much, until the Korath gathered a larger fleet. When they managed to injure one Archon, the others revealed frightening powers. They tore open rifts in reality, simply swallowing up Korath ships. Hidden super-weapons in planetoids, intended as mines, were harmless, as the Archons shattered the planetoids and atomized the weapons. In every battle, all Korath military ships were destroyed, but the civilians were somehow unscathed.`
			`	"The Korath saw certain defeat, and in surrender the Drak offered two options. The Korath may give up their religions and live as pets of their former friends, the Quarg. Or, they could keep their religions and live in starving exile near us.`
			`	"The Drak left the exiles with a warning: destroy Far'en Lei, and we will kill you all. You may never leave exile until you heal that planet and live there peacefully. As a precaution, and almost as a religion, they forbid weapons on that planet. A condition YOU violated." She looks heated for a moment before continuing.`

> 			`	"As to their guardianship over us, it violates the letter of the law, but not the intent. The Korath believe the Drak will look favorably upon them for protecting the weak. Their aid - at our insistence - has been purely civilian, except to defend us against major attacks. Anything we receive, the Syndicate may somehow obtain, and I shudder to think of what they may do with the wrong knowledge."`

Reviewer 2: ```suggestion
			`	"As to their guardianship over us, it violates the letter of the law, but not the intent. The Korath believe the Drak will look favorably upon them for protecting the weak. Along with defending us against major attacks, they have provided extensive assistance. Their aid - at our insistence - has been purely civilian: Anything we receive, the Syndicate may somehow obtain, and I shudder to think of what they may do with the wrong knowledge."`
```

Mostly just a flow thing, but also making it a bit clearer.
